src/aks-preview/HISTORY.rst

```diff
@@ -12,6 +12,10 @@ To release a new version, please select a new version number (usually plus 1 to
 Pending
 +++++++
 
+5.0.0b4
```

Comment:
just add more tests, maybe we don't need to bump the version @FumingZhang

Reply:
the last draft PR had blocking warning telling me to upgrade the version but if it is better to not I can revert.

Reply:
Agree with Charlie, don't need to release a new version if you're updating tests. The warning/comment could be ignored and only required CI checks are blocking.